nni/tools/nnictl/nnictl.py

```diff
@@ -115,6 +115,11 @@
 For your customized tuner/advisor, the file could have any json content depending on
 how you implement the corresponding methods (e.g., ``import_data``).
 
+.. note::
+
+    You can't see import data on the web portal when you import data to the experiment. Because currently the import data
```

Comment:
see imported data

import data into the experiment

Reply:
yes! will change it!